Dockerfile

```diff
@@ -1,4 +1,14 @@
-FROM mcneilco/acas-r-repo:1.13.7
+FROM mcneilco/acas-r-repo:2023.1.0
+
+# NODE
+USER root
+ENV NPM_CONFIG_LOGLEVEL warn
+ENV NODE_VERSION 14.x
```

Comment:
Shouldn't this say 18?

Reply:
Dang, yes.  I had looked at the ACAS repo forgetting that we had rolled back the node bump there.  I'll update it and re-test.

Reply:
Thanks, fixed.